homeassistant/components/knx/notify.py

```diff
@@ -28,16 +28,15 @@ async def async_get_service(
     if platform_config := hass.data[DATA_KNX_CONFIG].get(NotifySchema.PLATFORM):
         xknx: XKNX = hass.data[DOMAIN].xknx
 
-        notification_devices = []
-        for device_config in platform_config:
-            notification_devices.append(
-                XknxNotification(
-                    xknx,
-                    name=device_config[CONF_NAME],
-                    group_address=device_config[KNX_ADDRESS],
-                    value_type=device_config[CONF_TYPE],
-                )
+        notification_devices = [
+            XknxNotification(
+                xknx,
+                name=device_config[CONF_NAME],
+                group_address=device_config[KNX_ADDRESS],
+                value_type=device_config[CONF_TYPE],
             )
+            for device_config in platform_config
+        ]
         return (
             KNXNotificationService(notification_devices)
             if notification_devices
```

Comment:
@farmio Is this last if statement needed? On L28 we already check if platform_config is not falsey (and since it's a list this would confirm it has items in it) and thus notification_devices would never be empty right?

Reply:
I always need to REPL that if-walrus-get but yes, you are right 🙃